Fix in-breeze CLI tools to work also on Linux

potiuk opened the pull request:
This fixes CLI wrappers introduced in
f17a02d

The wrappers were not installed via Dockerfile and the ownership
fixing did not work on Linux

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

potiuk committed:
Fix in-breeze CLI tools to work also on Linux

Instead of creating the links in the image (which did not work)
the links are created now at the entry to the breeze image.

mik-laj commented on scripts/ci/run_cli_tool.sh:
```diff
@@ -157,12 +157,8 @@ RES=$?
-        "find ${DIRECTORIES_TO_FIX[@]@Q} -user root -print0 | xargs --null chown '${HOST_USER_ID}.${HOST_GROUP_ID}' --no-dereference")
-
-    docker run "${FIX_DOCKER_ARGS[@]}" "${AIRFLOW_CI_IMAGE}" "${FIX_COMMAND[@]}" >/dev/null 2>&1
+    find "${DIRECTORIES_TO_FIX[@]}" -user root -print0 2>/dev/null | \
```
Docker is needed here to have access to the volume with credentials

potiuk replied:
Correct. I simiplified it a bit and changed it to use the script we already had for that purpose. I also found out that we had another problem. "set +e" was missing so when the command failed, fixing ownership did not happen. I fixed that as well.

Pls. take a look @feluelle @mik-laj

potiuk committed:
fixup! Fix in-breeze CLI tools to work also on Linux

potiuk committed:
Fix in-breeze CLI tools to work also on Linux

This fixes CLI wrappers introduced in
f17a02d33047ebbfd9f92d3d1d54d6d810f596c1

The wrappers were not installed via Dockerfile and the ownership
fixing did not work on Linux
